scopesim/effects/electronic.py

```diff
@@ -215,12 +215,11 @@ def apply_to(self, obj, **kwargs):
             if dit < from_currsys(self.meta["mindit"]):
                 dit = from_currsys(self.meta["mindit"])
                 ndit = int(np.floor(exptime / dit))
-                print("Warning: The detector will be saturated!")
+                logger.warning("Warning: The detector will be saturated!")
```

Comment:
```suggestion
                logger.warning("The detector will be saturated!")
```
Otherwise we'll end up with `WARNING: Warning: The detector will be saturated!` in the current settings 🙃